.github/workflows/build.yml

```diff
@@ -39,6 +39,23 @@ jobs:
         run: make build-linux-amd64
       - name: Run Unit Tests
         run: make test
+      - name: Run E2E Tests
+        run: |
+          cd $GITHUB_WORKSPACE/test/e2e
+          go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo
+          tar -xvf testdata/distribution/mount.tar -C testdata/distribution/ > /dev/null
```

Comment:
We should do a gzip on the `mount.tar`. BTW< why are we pipe `stdout` to `/dev/null`? If we don't want the output, we can remove `v` in `xvf` where `v` means verbose.

Reply:
will keep verbose and rename mount.tar to docker.tar.gzip